### PR TITLE
Adding Users and Profiles test data

### LINKED ISSUE #48: Adding users test data
Adding user data when running `migrate` operation

### PR DESCRIPTION
**Pull request by MatanP12**
- Adding Users and Profiles test data to be created when using `migrate`
- Creating `superuser` account on migrate commend.

fix #48

### CODE REVIEW

**Review by AlonShakaroff — DISMISSED** [review]
LGTM

**Review by OmerCS8 — APPROVED** [review]
LGTM

**Review by natifridman — APPROVED** [review]

**Review by yulisuliman — APPROVED** [review]
Good job!

**Review by OmerCS8 — DISMISSED** [review]
Looks great!

**Comment by MatanP12** [comment]
Please do not merge the PR yet!

**Review by OmerCS8 — DISMISSED** [review]
Looks great, especially this Omer dude😉.

**Review by AlonShakaroff — APPROVED** [review]
LGTM

File: users/migrations/0002_add_users_test.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        superuser.email = 'admin@gmail.com'
+        superuser.set_password('admin1234')
+        superuser.save()
+        # Create normal users
+        users_test_data = [
+            ("AlonShakaroff", "Alon", "Shakaroff", "test$1234", "alon@gmail.com", "profile_pics/alon.jpg"),
+            ("OmerCohenShor", "Omer", "Cohen-Shor", "test$5678", "omer@gmail.com", "profile_pics/omer.jpg"),
+            ("TomerNewman", "Tomer", "Newman", "test$8910", "tomer@gmail.com", "profile_pics/tomer.jpeg"),
+            ("YuliSuliman", "Yuli", "Suliman", "test$3643", "yuli@gmail.com", "profile_pics/yuli.jpg"),
+            ("MatanPeretz", "Matan", "Peretz", "test$8979", "matan@gmail.com", "profile_pics/matan.jpg")
```
Comment (natifridman) [inline users/migrations/0002_add_users_test.py:27]: Is this the correct path? looks like the pics are under the media folder

Comment (MatanP12) [reply users/migrations/0002_add_users_test.py:27]: > Is this the correct path? looks like the pics are under the media folder

Yes, in the ResuMe/settings.py, the MEDIA_ROOT is the media directory.
image

**Review by natifridman — APPROVED** [review]
Approved, please squash your commits

**Review by AlonShakaroff — APPROVED** [review]

**Review by TomerNewmanPrograms — APPROVED** [review]
LGTM

**Review by OmerCS8 — APPROVED** [review]

**Review by natifridman — APPROVED** [review]

**Pull request merged**